Added docs fragment for proxysql

bmildren opened the pull request:
##### SUMMARY
Adding a documentation fragment for the proxysql modules.

##### ISSUE TYPE
 - Docs Pull Request

##### COMPONENT NAME
http://docs.ansible.com/ansible/latest/list_of_database_modules.html#proxysql

##### ANSIBLE VERSION
```
ansible 2.5.0 (add_docs_fragment 6f9ed4cc28) last updated 2017/11/02 15:30:54 (GMT +100)
  config file = None
  configured module search path = [u'/Users/ben/.ansible/plugins/modules', u'/usr/share/ansible/plugins/modules']
  ansible python module location = /Users/ben/Work/bmildren/ansible/lib/ansible
  executable location = /Users/ben/Work/bmildren/ansible/bin/ansible
  python version = 2.7.12 (default, Oct 11 2016, 05:20:59) [GCC 4.2.1 Compatible Apple LLVM 8.0.0 (clang-800.0.38)]
```


##### ADDITIONAL INFORMATION
I've verified the change locally using ansible-docs, adding docs fragment here is a precursary step, I'll create a separate PR for each individual module to update and extend the existing docs.

bmildren committed:
Added docs fragment for proxysql

gundalow commented on lib/ansible/utils/module_docs_fragments/proxysql.py:
```diff
@@ -0,0 +1,47 @@
+    default: 6032
+  config_file:
+    description:
+      - Specify a config file from which login_user and login_password are to
```
`I(login_user)` `I(login_password)`

nitzmahone commented:
@bmildren feel free to make all the necessary changes in this PR- we'd rather have it all land in one commit than a bunch of individual PRs

gundalow commented on lib/ansible/utils/module_docs_fragments/proxysql.py:
```diff
@@ -0,0 +1,47 @@
+      - Save mysql host config to sqlite db on disk to persist the
+        configuration.
+    default: True
+  load_to_runtime:
```
`type: bool`?

bmildren committed:
Added formatting for option names, and bool types

bmildren committed:
Updated proxysql modules to use doc fragments

bmildren commented:
Thanks for the feedback!

@nitzmahone I've updated and included the individual modules in the PR.

@gundalow I've added the bool types, and the formatting for references to the module options.

bmildren commented:
Thanks for the feedback!

@nitzmahone I've updated the PR to include the changes that update the individual modules to use the new doc fragments.

@gundalow I've added the bool types, and the formatting for the references to the modules options.

Thanks! Ben

bmildren commented:
Hi @gundalow 

Can you confirm if the changes made have addressed those you requested?

Thanks! Ben

gundalow commented:
Merged into `devel`, will be released in Ansible 2.5.0
Thanks for tidying this up
